From fc733b5212f56597633e19a6831f8c8653884e38 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Wed, 04 Jul 2018 15:56:41 +0300
Subject: [PATCH] board: habana: switch remote power to pdu

Switch the default for remote power to use the pdu.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/board/habanalabs/common/remote_power b/board/habanalabs/common/remote_power
index 8900eeb..8b0f25a 100755
--- a/board/habanalabs/common/remote_power
+++ b/board/habanalabs/common/remote_power
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-power=t32_viv
-#power=pdu
+#power=t32_viv
+power=pdu
 
 cd `dirname $0`
 
